nosqlbench/http-jsonapi-crud-basic.yaml

```diff
@@ -1,4 +1,4 @@
-min_version: "5.17.1"
+min_version: "5.17.3"
 
 # nb5 -v run driver=http yaml=http-jsonapi-crud-basic jsonapi_host=my_jsonapi_host auth_token=$AUTH_TOKEN
```

Comment:
* we should also probably update this and remove the `auth_token` now as it seems invalid
* is driver needed at all?
* furthermore, why not having localhost here as default hostname
* and maybe also have `docscount=` and `connections=` in parameters

Just in order I can copy this line and run locally..